shell/packages/sandstorm-db/db.js

```diff
@@ -1810,23 +1810,81 @@ if (Meteor.isServer) {
   // Cleanup tokens every hour.
   SandstormDb.periodicCleanup(3600000, cleanupExpiredAssetUploads);
 
-  SandstormDb.prototype.getIconUrls = function (icon) {
-    check(icon, Match.OneOf(undefined, null,
-                            { format: String, assetId: String,
-                              assetId2xDpi: Match.Optional(String), }));
-    const root =
-        Url.parse(process.env.ROOT_URL).protocol + "//" + this.makeWildcardHost("static") + "/";
+  SandstormDb.prototype.updateCachedViewInfo = function (grainId, viewInfo) {
+    check(grainId, String);
+    // TODO(soon): Typecheck viewInfo. We should be able to factor out some of the checking code
```

Comment:
I guess type checking viewInfo isn't critical because we normally receive it via Cap'n Proto, which already enforces the type, right? Mention that here.